ingestion/src/metadata/orm_profiler/orm/converter.py

```diff
@@ -100,12 +101,54 @@ def ometa_to_orm(
     }
 
     # Type takes positional arguments in the form of (name, bases, dict)
-    return type(
+    orm = type(
         table.fullyQualifiedName.replace(".", "_"),  # Output class name
         (Base,),  # SQLAlchemy declarative base
         {
             "__tablename__": str(table.name.__root__),
-            "__table_args__": {"schema": str(database.name.__root__)},
+            "__table_args__": {
+                "schema": get_db_name(database),
+                "extend_existing": True,  # Recreates the table ORM object if it already exists. Useful for testing
+            },
             **cols,
         },
     )
+
+    if not isinstance(orm, DeclarativeMeta):
+        raise ValueError("OMeta to ORM did not create a DeclarativeMeta")
```

Comment:
Helps us properly type `DeclarativeMeta`. Otherwise the `type` creation of a class def was not properly handling resulting types